onadata/libs/serializers/project_serializer.py

```diff
@@ -1,35 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+Project Serializer module.
+"""
 from future.utils import listvalues
 
-from rest_framework import serializers
-from django.db.utils import IntegrityError
 from django.conf import settings
 from django.contrib.auth.models import User
 from django.core.cache import cache
+from django.db.utils import IntegrityError
 from django.utils.translation import ugettext as _
 
-from onadata.apps.logger.models import Project
-from onadata.apps.logger.models import XForm
-from onadata.libs.permissions import OwnerRole
-from onadata.libs.permissions import ReadOnlyRole
-from onadata.libs.permissions import is_organization
-from onadata.libs.permissions import get_role
+from rest_framework import serializers
+
+from onadata.apps.api.models import OrganizationProfile
+from onadata.apps.api.tools import (get_organization_members_team,
+                                    get_organization_owners_team)
+from onadata.apps.logger.models import Project, XForm
+from onadata.libs.permissions import (OwnerRole, ReadOnlyRole, get_role,
+                                      is_organization)
+from onadata.libs.serializers.dataview_serializer import \
+    DataViewMinimalSerializer
 from onadata.libs.serializers.fields.json_field import JsonField
 from onadata.libs.serializers.tag_list_serializer import TagListSerializer
-from onadata.libs.serializers.dataview_serializer import (
-    DataViewMinimalSerializer
-)
-from onadata.libs.utils.decorators import check_obj
 from onadata.libs.utils.cache_tools import (
-    PROJ_FORMS_CACHE, PROJ_NUM_DATASET_CACHE, PROJ_PERM_CACHE,
-    PROJ_SUB_DATE_CACHE, safe_delete, PROJ_TEAM_USERS_CACHE,
-    PROJECT_LINKED_DATAVIEWS, PROJ_BASE_FORMS_CACHE)
-from onadata.apps.api.tools import (
-    get_organization_members_team, get_organization_owners_team)
+    PROJ_BASE_FORMS_CACHE, PROJ_FORMS_CACHE, PROJ_NUM_DATASET_CACHE,
+    PROJ_PERM_CACHE, PROJ_SUB_DATE_CACHE, PROJ_TEAM_USERS_CACHE,
+    PROJECT_LINKED_DATAVIEWS, safe_delete)
+from onadata.libs.utils.decorators import check_obj
 
 
 def get_obj_xforms(obj):
```

Comment:
I don't remember if there's a historical reason why we have `obj` throughout this namespace, but it looks like a future refactor would be to change that to `project`